.github/workflows/create_version_bump_PR.yml

```diff
@@ -64,17 +82,17 @@ jobs:
           sed -i -e "s/DEVELOPMENT_VERSION=\"$CURRENT_DEV_VERSION/DEVELOPMENT_VERSION=\"$DEV_VERSION/g" dotnet/build-packages.sh
           echo Replaced dev version in file.
 
-          PR_BODY="Bumps the release version to $VERSION."
+          PR_BODY="Bumps the release version to $VERSION"
 
-          TRACER_VERSION=${{ github.event.inputs.tracer_version }}
+          TRACER_VERSION=${{steps.input.outputs.tracer_version}}
```

Comment:
gotta love  the naming of `steps.input.outputs` 😂

Reply:
Maybe the `input` step could be renamed to `set_variables` so it's not so confusing, but not a blocker